component/tests/golden/vshn/appcat/appcat/21_composition_vshn_postgres.yaml

```diff
@@ -1001,22 +1003,21 @@ spec:
                         for: 1h
                         labels:
                           severity: warning
-                  - name: postgresql-memory
-                    rules:
                       - alert: PostgreSQLMemoryCritical
                         annotations:
                           description: |-
-                            The memory claimed by the instance {{ $labels.name }} in namespace {{ $labels.label_appcat_vshn_io_claim_namespace }} has been over 85% for 2 hours.
```

Comment:
Why did the value change from 85% to 90%?

Reply:
I changed it back to 85%